### PR TITLE
CAL-209: updated NSILI for the latest versioning changes

### PR DESCRIPTION
**Pull request by rzwiefel**
#### What does this PR do?
Fixes Incompatibilities with latest versioning changes
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@bdeining
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@jlcsmith
@kcwire

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -37,7 +37,11 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
+<<<<<<< HEAD
```
Comment (jlcsmith) [inline pom.xml]: looks like a merge issue here @rzwiefel

Comment (rzwiefel) [inline pom.xml]: Fixed

**Comment by glenhein** [comment]
I built ddf master, then build this CAL-209 branch. I talked to @rzwiefel about the best way to test the change. Unfortunately, he didn't know, and recommended to just make sure it builds. So, I built, installed, and ingested a random piece of content. That's not a great hero, but if someone has a better test, let me know.

**Comment by jlcsmith** [comment]
@troymohl probably can answer the testing question best.  Itests may cover this but not 100% sure.

File: catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/BqsConverter.java [hunk 2]
```diff
@@ -784,7 +784,7 @@ public void exitQuoted_string(BqsParser.Quoted_stringContext ctx) {
             Filter filter = null;
             if (StringUtils.isNotBlank(attribute)) {
                 if (attribute.equals(MetacardVersion.ACTION)) {
-                    if (stringValue.equals(NsiliCardStatus.NEW)) {
+                    if (stringValue.equals(NsiliCardStatus.CHANGED)) {
```
Comment (jlcsmith) [inline catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/BqsConverter.java]: @rzwiefel can you answer this?  I believe that is correct.

Comment (rzwiefel) [inline catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/BqsConverter.java]: Not a clue. I just cherry-picked the commit :P

Comment (jlcsmith) [inline catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/BqsConverter.java]: Duh - that should have been a question for @troymohl

Comment (troymohl) [inline catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/BqsConverter.java]: I thought we only had history on CHANGED/DELETED cards and wouldn't have a history card for NEW anymore. So if we don't have a history card, then we should mark it new? I thought thats what these changes were doing, but maybe I missed something.

For NSILI, they have NEW, CHANGED, OBSOLETE.

Comment (jlcsmith) [inline catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/BqsConverter.java]: That sounds correct to me Troy.

**Comment by troymohl** [comment]
@jlcsmith the various unit tests should cover these cases. There may not be a specific one for BqsConverter though that only tests the status. In addition to BqsConverterTest ResultDAGConverterTest also has some tests for versioned content.

**Pull request merged**